bundles/org.openhab.binding.edf/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="edf" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="base">
+		<label>Base Tariff</label>
+		<description>Default Tariff Option</description>
+
+		<channels>
+			<channel id="tariff-start" typeId="timestamp"/>
+			<channel id="fixed-ht" typeId="price-ht">
+				<label>Fix Price HT</label>
+				<description>Yearly fix price excluding taxes.</description>
```

Comment:
Probably rather "fixed" than "fix" ?

Same for fixed-ttc

Reply:
Changed.

Reply:
Please also change the label field

Reply:
Done

Reply:
Please also changed in README tables.

Reply:
Sir, yes sir ! Done